Can't build with Swift 5.1 with SwiftPM

# Report

## What did you do?

1. Clone Down
2. `xcode-select` Xcode 11 beta 1
3. `swift build`

## What did you expect to happen?

It should build.

## What happened instead?

```
» swift build
In file included from /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/scanners.c:3:
In file included from /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/chunk.h:7:
/Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/cmark.h:5:10: error: 'cmark_export.h' file not found with <angled> include; use "quotes" instead
#include <cmark_export.h>
         ^~~~~~~~~~~~~~~~
         "cmark_export.h"
/Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/cmark.h:6:10: error: 'cmark_version.h' file not found with <angled> include; use "quotes" instead
#include <cmark_version.h>
         ^~~~~~~~~~~~~~~~~
         "cmark_version.h"
.
[0/19] Compiling libcmark xml.c
```

This also affects building Down as a SwiftPM package using the new support in Xcode 11.

> Happy to accept a PR for this, but personally not using SwiftPM

I totally understand. I opened it in case anyone watching was interested in fixing it, but I'll definitely try to come back around when I get a chance if no one beats me to it. (And I'm also not sure how to fix this one.)

> Can you output what happens when you run `swift build -v`.

```
xcrun --sdk macosx --show-sdk-path
xcrun --sdk macosx --show-sdk-platform-path
xcrun --sdk macosx --find xctest
sandbox-exec -p '(version 1)
(deny default)
(import "system.sb")
(allow file-read*)
(allow process*)
(allow sysctl*)
(allow file-write*
    (regex #"^/private/var/tmp/org\.llvm\.clang.*")
    (regex #"^/var/folders/t1/0hp_mdnj14qbqb41mc221qkw0000gn/T/org\.llvm\.clang.*")
    (regex #"^/private/var/folders/t1/0hp_mdnj14qbqb41mc221qkw0000gn/T/org\.llvm\.clang.*")
    (regex #"^/private/var/folders/t1/0hp_mdnj14qbqb41mc221qkw0000gn/C/org\.llvm\.clang.*")
    (subpath "/Users/mdiep/Repositories/iwasrobbed/Down/.build")
)
' /Applications/Xcode11.0.0b1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc --driver-mode=swift -L /Applications/Xcode11.0.0b1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/4_2 -lPackageDescription -swift-version 5 -I /Applications/Xcode11.0.0b1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/4_2 -target x86_64-apple-macosx10.10 -sdk /Applications/Xcode11.0.0b1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk /Users/mdiep/Repositories/iwasrobbed/Down/Package.swift -fileno 7
xcrun --find clang
/Applications/Xcode11.0.0b1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -fobjc-arc -target x86_64-apple-macosx10.11 -isysroot /Applications/Xcode11.0.0b1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -F /Applications/Xcode11.0.0b1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks -g -O0 -DSWIFT_PACKAGE=1 -DDEBUG=1 -fblocks -index-store-path /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/index/store -fmodules -fmodule-name=libcmark -I /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/include -fmodules-cache-path=/Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/ModuleCache -MD -MT dependencies -MF /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/libcmark.build/utf8.c.d -c /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/utf8.c -o /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/libcmark.build/utf8.c.o
/Applications/Xcode11.0.0b1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -fobjc-arc -target x86_64-apple-macosx10.11 -isysroot /Applications/Xcode11.0.0b1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -F /Applications/Xcode11.0.0b1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks -g -O0 -DSWIFT_PACKAGE=1 -DDEBUG=1 -fblocks -index-store-path /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/index/store -fmodules -fmodule-name=libcmark -I /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/include -fmodules-cache-path=/Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/ModuleCache -MD -MT dependencies -MF /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/libcmark.build/scanners.c.d -c /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/scanners.c -o /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/libcmark.build/scanners.c.o
/Applications/Xcode11.0.0b1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -fobjc-arc -target x86_64-apple-macosx10.11 -isysroot /Applications/Xcode11.0.0b1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -F /Applications/Xcode11.0.0b1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks -g -O0 -DSWIFT_PACKAGE=1 -DDEBUG=1 -fblocks -index-store-path /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/index/store -fmodules -fmodule-name=libcmark -I /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/include -fmodules-cache-path=/Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/ModuleCache -MD -MT dependencies -MF /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/libcmark.build/xml.c.d -c /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/xml.c -o /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/libcmark.build/xml.c.o
/Applications/Xcode11.0.0b1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -fobjc-arc -target x86_64-apple-macosx10.11 -isysroot /Applications/Xcode11.0.0b1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -F /Applications/Xcode11.0.0b1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks -g -O0 -DSWIFT_PACKAGE=1 -DDEBUG=1 -fblocks -index-store-path /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/index/store -fmodules -fmodule-name=libcmark -I /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/include -fmodules-cache-path=/Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/ModuleCache -MD -MT dependencies -MF /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/libcmark.build/render.c.d -c /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/render.c -o /Users/mdiep/Repositories/iwasrobbed/Down/.build/x86_64-apple-macosx/debug/libcmark.build/render.c.o
In file included from /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/render.c:2:
In file included from /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/buffer.h:10:
/Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/cmark.h:5:10: error: 'cmark_export.h' file not found with <angled> include; use "quotes" instead
#include <cmark_export.h>
         ^~~~~~~~~~~~~~~~
         "cmark_export.h"
/Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/cmark.h:6:10: error: 'cmark_version.h' file not found with <angled> include; use "quotes" instead
#include <cmark_version.h>
         ^~~~~~~~~~~~~~~~~
         "cmark_version.h"
In file included from /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/xml.c:7:
/Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/cmark.h:5:10: error: 'cmark_export.h' file not found with <angled> include; use "quotes" instead
#include <cmark_export.h>
         ^~~~~~~~~~~~~~~~
         "cmark_export.h"
In file included from /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/scanners.c:3:
In file included from /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/chunk.h:7:
/Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/cmark.h:5:10: error: 'cmark_export.h' file not found with <angled> include; use "quotes" instead
#include <cmark_export.h>
         ^~~~~~~~~~~~~~~~
         "cmark_export.h"
/Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/cmark.h:6:10: error: 'cmark_version.h' file not found with <angled> include; use "quotes" instead
#include <cmark_version.h>
         ^~~~~~~~~~~~~~~~~
         "cmark_version.h"
/Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/cmark.h:6:10: error: 'cmark_version.h' file not found with <angled> include; use "quotes" instead
#include <cmark_version.h>
         ^~~~~~~~~~~~~~~~~
         "cmark_version.h"
In file included from /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/utf8.c:6:
In file included from /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/utf8.h:5:
In file included from /Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/buffer.h:10:
/Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/cmark.h:5:10: error: 'cmark_export.h' file not found with <angled> include; use "quotes" instead
#include <cmark_export.h>
         ^~~~~~~~~~~~~~~~
         "cmark_export.h"
/Users/mdiep/Repositories/iwasrobbed/Down/Source/cmark/cmark.h:6:10: error: 'cmark_version.h' file not found with <angled> include; use "quotes" instead
#include <cmark_version.h>
         ^~~~~~~~~~~~~~~~~
         "cmark_version.h"
2 errors generated.
2 errors generated.
2 errors generated.
```

--- a/Package.swift
+++ b/Package.swift
@@ -20,7 +20,9 @@ let package = Package(
         .target(
             name: "libcmark",
             dependencies: [],
-            path: "Source/cmark"),
+            path: "Source/cmark",
+            exclude: ["include"],
+            publicHeadersPath: "./"),
         .target(
             name: "Down",
             dependencies: ["libcmark"],

